renovate-config/kustomization.yaml

```
@@ -0,0 +1,6 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+configMapGenerator:
+- name: renovate-config
+  files:
+  - renovate.json
```

```suggestion
  files:
  - renovate.json
  options:
    disableNameSuffixHash: true
```
Without disabling name suffix, the configmap will be created with name `renovate-config-c7mff5kd49` which has a hash suffix, MintMaker only looks for configmap `renovate-config`.

You check that by running:
```
$ kustomize build https://github.com/querti/mintmaker-config/renovate-config?ref=acf9a152cde1a40b96b652ba58cbd39726781919
```

---

Good point, added, thanks.